things/postgres/init.go

```diff
@@ -93,6 +93,29 @@ func migrateDB(db *sqlx.DB) error {
 					 metadata TYPE JSONB using metadata::text::jsonb`,
 				},
 			},
+			{
+				Id: "things_4",
+				Up: []string{
+					`ALTER TABLE IF EXISTS things ADD CONSTRAINT things_id_key UNIQUE (id)`,
+					`CREATE TABLE IF NOT EXISTS thing_groups ( 
+						id          UUID NOT NULL,
+						parent_id   UUID, 
+						owner_id    UUID,
+						name        VARCHAR(254) UNIQUE NOT NULL,
+						description VARCHAR(1024),
+						metadata    JSONB,
+						PRIMARY KEY (id),
+						FOREIGN KEY (parent_id) REFERENCES thing_groups (id)  ON DELETE CASCADE ON UPDATE CASCADE
```

Comment:
Please check the formatting, it looks like there are multiple whitespaces in some lines that start with `FOREIGN_KEY`.